10_binary_counter.md

```diff
@@ -253,43 +254,40 @@ we just rebalanced parenthesis[^min-not-commutative].
 
         f(a, b) = f(b, a)
 
-    Informally, `f` does the same thing, regardless of the order of the inputs.
+    Informally, `f` gets the same result, regardless of the order of the inputs.
     For example, multiplication of integers is commutative:
 
         a * b = b * a
 
-    This fact about integers can be proven in the following manner:
-    If you arrange items into a block of columns and rows, such as: 
-
-        * * * * *
-        * * * * *
-        * * * * *
+    In Chapter 9.1 of "From Mathematics to Generic Programming", Alex gives
+    a neat visual proof of the commutativity of positive integers:
```

Comment:
So I think this should read
```suggestion
    a neat visual proof of the commutativity of multiplication over positive integers:
```

Reply:
Thanks. I didn't also I didn't include your language about base * width of rectangles because mixing the discrete integers case and real numbers/area of shapes gets theortically complex very quickly. I think the picture with the rows/columns of soldiers is sufficient.